owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -378,16 +365,14 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                 pRecommend.setOnPreferenceClickListener(preference -> {
 
                     Intent intent = new Intent(Intent.ACTION_SENDTO);
-                    intent.setType("text/plain");
                     intent.setData(Uri.parse(getString(R.string.mail_recommend)));
                     intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
 
                     String appName = getString(R.string.app_name);
                     String downloadUrl = getString(R.string.url_app_download);
 
                     String recommendSubject =
```

Comment:
This fits properly in the same line.

Reply:
This is pending as well @hannesa2

Reply:
<img width="834" alt="image" src="https://user-images.githubusercontent.com/3314607/50663492-1dc7c880-0faa-11e9-828f-691d7ff70e99.png">

Reply:
In this class more of them are marked as lint issues

Reply:
It's vice versa, it was 3 lines, now it's only one !
<img width="817" alt="image" src="https://user-images.githubusercontent.com/3314607/50685244-b6952d00-1018-11e9-9b6c-b82455566bdc.png">

Like https://github.com/owncloud/android-library/pull/224 this repo needs this too